sympy/physics/vector/frame.py

```diff
@@ -190,9 +190,9 @@ def __init__(self, name, indices=None, latexs=None, variables=None):
             self.latex_vecs = latexs
         self.name = name
         self._var_dict = {}
-        #The _dcm_dict dictionary will only store the dcms of parent-child
-        #relationships. The _dcm_cache dictionary will work as the dcm
-        #cache.
+        #The _dcm_dict dictionary will only store the dcms of immediate parent-child
+        #relationships. The _dcm_cache dictionary will store calculated dcm along with
+        #all content of _dcm_dict for faster retrieval of dcms.
```

Comment:
I still don't understand this last sentence. What is different in _dcm_cache than _dcm_dict?

Reply:
@moorepants  `_dcm_cache` stores everything inside `_dcm_dict` but also stores calculated `dcm` value of that frame.
If we do something like
```
A.oreint(B)
B.orient(C)
```
then for frame C `_dcm_dict `and `_dcm_cache` would have nothing related to frame B, but we do havee a connecting path between A and C.

And now if we do
```
C.dcm(A) or A.dcm(A)
```
then this calculated dcm would be stored in `_dcm_cache` and won't effect `_dcm_dict`.